config/doxygen_config.in

```diff
@@ -2166,7 +2166,8 @@ EXTERNAL_PAGES         = YES
 # interpreter (i.e. the result of 'which perl').
 # The default file (with absolute path) is: /usr/bin/perl.
 
-PERL_PATH              = /usr/bin/perl
+# PERL_PATH              = /usr/bin/perl
+@PERL_PATH@
```

Comment:
Is this change related to spack, or is there some other reason we might expect the PERL_PATH to be something other than usr/bin/perl?

Reply:
The `PERL_PATH` input is deprecated by newer doxygen versions.  This entry causes an error if left in place.